### ISSUE TITLE
Question about running the model.

### ISSUE THREAD

**Issue by frankkim1108** [zhenliuZJU/EVA-Gaussian#3]
Do I have to change any other parameters in the train yaml file if I want to change the batch size to 1? 
ex) learning rate, weight decay, step number....

Also, could you share the evaluation code?
Which model did you use to calculate LPIPS? VGG, ALEX, Squeeze?

Thank you for you time.

**Comment by zhenliuZJU** [comment]
Thank for your interest.

Yes, you need to modify the hyperparameters if you change the batch size. You may try to increase the lr or step number to see if it can remain the accuracy.

As for evaluation, we adopt the most commonly used VGG model to calculate LPIPS. 
Here is an example:
```
import lpips as lp

lp_vgg = lp.LPIPS(net='vgg').cuda()
lpips = lp_vgg(img1_, img2_)
```
